app/Console/Commands/Cloud/CloudDeleteUser.php

```
@@ -54,124 +55,141 @@ public function handle()
-                }
+                        return 1;
+                    }
 
-                // Phase 4: Handle Teams
-                if (! $this->handleTeams()) {
-                    DB::rollBack();
-                    $this->error('User deletion failed at team handling phase. All changes rolled back.');
+                    // Phase 5: Cancel Stripe Subscriptions
+                    if (! $this->skipStripe && isCloud()) {
+                        if (! $this->cancelStripeSubscriptions()) {
+                            DB::rollBack();
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'handle'. (undefined)

(StaticAccess)